app/javascript/views/usage/index.js

```diff
@@ -2,6 +2,24 @@ import { isObject } from '../../utils/isType';
 import { initializeCharts, createChart, drawHorizontalBar } from '../../utils/charts';
 
 $(() => {
+  // fns to handle the separator character menu
+  // for CSV download
+  const changeStatFnGen = (str) => {
+    const fn = (item, index) => {
+      item.href = item.href.replace(/sep=.*/, str);
+    };
+    return fn;
+  };
+
+  // attach listener to separator select menu
+  // on change look for "stat" elements and chnage their query param
+  document.getElementById('csv-field-sep').addEventListener('click', (e) => {
```

Comment:
we should look into converting this over to a `<form>` submission instead of a `<a>` link, that way we wouldn't need to do any manual JS it would just come through in the submission params.

Something to do after the Rails 5 upgrade